project/plugins.sbt

```diff
@@ -5,8 +5,6 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.1.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
```

Comment:
According to https://confluence.tools.tax.service.gov.uk/display/TEC/2021/05/05/New+version+of+sbt-auto-build
`sbt-git-versioning` is provided transitively, so it can be removed from your project's plugins.sbt.

Reply:
👍